ui/src/app/utils/access-utils.tsx

```diff
@@ -476,7 +485,7 @@ export const computeRenewalDisplayDates = (
       }),
     ],
     [
-      isExpiring(expirationEpochMillis),
+      isExpiringOrExpired(expirationEpochMillis),
```

Comment:
why orExpired? because if getWholeDaysFromNow(expiration) = serverConfigStore.get().config.accessRenewalLookback it is considered as expired?